File: yixing-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.findPage = function (page,rows) {
        return $http.get("../brand/findPage.do?page=" + page + "&rows=" + rows);
    }

    this.save = function (methodName,pojo) {
        return $http.post('../brand/' + methodName + '.do',pojo);
    }

    this.findOne = function (id) {
        return $http.get("../brand/findById.do?id=" + id);
    }

    this.del = function (selectIds) {
        return $http.get("../brand/remove.do?ids=" + selectIds);
    }

    this.search = function (page,rows,searchEntity) {
       return  $http.post("../brand/search.do?page=" + page + "&rows=" + rows,searchEntity);
    }

    this.selectOptionList = function (){
        return $http.get("../brand/selectOptionList.do");
    }
});

File: yixing-sellergoods-interface/src/main/java/com/yixing/sellergoods/service/BrandService.java
package com.yixing.sellergoods.service;

import com.yixing.entity.PageResult;
import com.yixing.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    /**
     * 返回分页列表
     */
    public PageResult findPage(int pageNum,int pageSize);

    /**
     * 添加数据
     * @param brand
     */
    public void add(TbBrand brand);

    /**
     * 根据id查询品牌数据
     * @param id
     * @return
     */
    public TbBrand findById(long id);

    /**
     * 修改品牌数据
     * @param tbBrand
     */
    public void update(TbBrand tbBrand);

    /**
     * 根据id删除数据
     * @param ids
     */
    public void remove(long[] ids);

    public PageResult findPage(TbBrand brand,int pageNum,int pageSize);

    List<Map> selectOptionList();
}
